specifications/api/swagger.yaml

```diff
@@ -1319,6 +1319,20 @@ paths:
         - *parametersSortOrder
       responses: *responsesListTransactions
 
+  /byron-wallets/{walletId}/transactions/{transactionId}:
+    delete:
+      operationId: deleteByronTransaction
+      tags: ["Byron Transactions"]
+      summary: Delete
+      description: |
+        <p align="right">status: <strong>under development</strong></p>
```

Comment:
Should we try to be consistent and say "under testing" ?

Reply:
corrected